lib/util/cache/repository/impl/local.ts

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs-extra';
```

Comment:
why we don't use our fs layer? 🤔

Reply:
I'm not sure but think it's because we store files inside `cacheDir` and not the `localDir`, under which most `util/fs` functions are working

Reply:
There are two sets of functions, thew `local` ones are working on `localDir`, but the normal ones are mainly wrapper to `fs-extra` for easier test mocking.

So i prefer all code paths should go through our fs layer if possible.